common/config/rush/command-line.json

```diff
@@ -58,8 +57,7 @@
       "summary": "Check types in typed projects",
       "commandKind": "bulk",
       "ignoreMissingScript": true,
-      "enableParallelism": true,
-      "allowWarningsInSuccessfulBuild": true
```

Comment:
didn't need this even when originally added, no warnings emitted by type-check command